Rubrik/Rubrik.psd1

```diff
@@ -134,6 +134,7 @@
                    'Get-RubrikFilesetTemplate',
                    'Get-RubrikGuestOSCredential',
                    'Get-RubrikHost',
+                   'Get-RubrikHostVolumes',
```

Comment:
Need to have Pester Unit Tests created for both Get-RubrikHostVolume and Protect-RubrikVolumeGroup.  More than happy to help/walk you through how to do that if you like - but we aren't merging any new functionality into the module without required unit tests.